<?xml version="1.0"?>
<launch>
    <include file="$(find sot_youbot)/launch/pr2/upload_pr2.launch"/>

    <arg name="gui" default="False" />
    <param name="use_gui" value="$(arg gui)"/>
    <param name="robot_base_link" value="base_footprint"/>
    <!--<param name="ref_link" value="odom_combined"/>-->

    <arg name="frame" value="odom"/>
    <arg name="child_frame" value= "base_footprint"/>
<param name="ref_link" value="odom_combined"/>

  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" ></node>

    <node name="robot_pose_publisher" pkg="dynamic_graph_bridge" type="robot_pose_publisher" args="_frame:=$(arg frame) _child_frame:=$(arg child_frame)">
        <env name="ROS_NAMESPACE" value="/dynamic_graph" />
    </node>

     <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" >
	<env name="ROS_NAMESPACE" value="/dynamic_graph"/>
    </node>


    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find sot_pr2)/launch/pr2.rviz" />

</launch>

